yogstation/code/modules/clothing/shoes/colour.dm

```diff
@@ -0,0 +1,3 @@
+/obj/item/clothing/shoes/sneakers/white/pointed_at(var/mob/user)
```

Comment:
You don’t need the var/ part, and pointed_at(mob/user) looks cooler

Reply:
Don't question the chesse20 code, it's perfect like this